eng/common/pipelines/templates/steps/docindex.yml

```diff
@@ -49,6 +47,7 @@ jobs:
           targetPath: $(Build.ArtifactStagingDirectory)/docfx_project/_site
           
       - pwsh: |
+          git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" commit -am "Save the language updates."
```

Comment:
Do they have to be committed? Why not just revert them since you already made a copy of them.

Reply:
I can go with `git reset --hard HEAD`